src/Validation/Validator.php

```diff
@@ -1619,6 +1619,24 @@ public function isArray($field, $message = null, $when = null)
     }
 
     /**
+     * Add a validation rule to ensure that a field contains a scalar.
```

Comment:
Validation rules are still high level and for a user just seeing that rule it might still be valuable not having to check the php internal manuals maybe :)